features/nested_tuples.feature

```diff
@@ -0,0 +1,28 @@
+Feature: Nested Tuples
+
+  Scenario: Nested Tuple 1
```

Comment:
C# will execute the scenarios alphabetically
Maybe rename to '1 Nested tuple', '2 Nested Tuple', '3 Nested Tuple' ?